### ISSUE TITLE
Validate after Scaffold Resource throws 2 errors for searchOrCreates

### ISSUE THREAD

**Issue by abstractcoder** [zapier/zapier-platform-cli#47]
After using scaffold to create a new resource, I get the following validation errors:

```
┌─────────────┬────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ Property    │ App.searchOrCreates.clientSearchOrCreate.display.description                                           │
│ Message     │ does not meet minimum length of 12                                                                     │
│ Links       │                                                                                                        │
│                                                                                                                      │
│ Property    │ App.searchOrCreates.clientSearchOrCreate.key                                                           │
│ Message     │ must match a "key" from a search (options: clientSearch)                                               │
│ Links       │ https://github.com/zapier/zapier-platform-schema/blob/v1.0.3/docs/build/schema.md#searchorcreateschema │
└─────────────┴────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

The only way I can get the app to pass validation is to remove the search method from the client resource export. I don't see a way to specify the required description.

**Comment by FokkeZB** [comment]
Thanks for reporting @abstractcoder, and for sharing the code (via HS).

I can reproduce this in a new project:

```
mkdir horse && cd horse && zapier init . && npm i && zapier validate && zapier scaffold resource horse && zapier validate
```

Gives:

```
┌─────────────┬────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ Property    │ App.searchOrCreates.horseSearchOrCreate.display.description                                            │
│ Message     │ does not meet minimum length of 12                                                                     │
│ Links       │                                                                                                        │
│                                                                                                                      │
│ Property    │ App.searchOrCreates.horseSearchOrCreate.key                                                            │
│ Message     │ must match a "key" from a search (options: horseSearch)                                                │
│ Links       │ https://github.com/zapier/zapier-platform-schema/blob/v1.0.5/docs/build/schema.md#searchorcreateschema │
└─────────────┴────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

And indeed when I intercept [the schema](https://gist.github.com/FokkeZB/587e4d88270762ccd3476d97cd27e7c8) it shows that [`App.searchOrCreates.horseSearchOrCreate.display.description` is `""`](https://gist.github.com/FokkeZB/587e4d88270762ccd3476d97cd27e7c8#file-definition-json-L140):

```json
  "searchOrCreates": {
    "horseSearchOrCreate": {
      "key": "horseSearchOrCreate",
      "display": {
        "label": "Find or Create Horse",
        "description": ""
      },
      "search": "horseSearch",
      "create": "horseCreate"
    }
  }
```

I don't get the other error though. As I read the refered https://github.com/zapier/zapier-platform-schema/blob/v1.0.3/docs/build/schema.md#searchorcreateschema `App.searchOrCreates.horseSearchOrCreate.key` should be a unique key, not match a key from search as the error says.

When I copy the above to `index.js` I can fix the first error by manually setting a description, but changing the key to `horseSearch` doesn't fix the second one.

**Comment by BrunoBernardino** [comment]
Will be fixed in 1.0.10
